lib/atom_utils.dart

```diff
@@ -40,7 +40,9 @@ Future<String> which(String execName, {bool isBatchScript: false}) {
       return result;
     });
   } else {
-    return exec('which', [execName]).then((String result) {
+    if (_shellWrangler == null) _shellWrangler = new ShellWrangler();
```

Comment:
This looks identical to the (isMac) block above... combine with block above? if (!isWindows) ?